### PR TITLE
[chore] Update of various bits of info

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Just a few trivial updates.

### CODE REVIEW

**Comment by mbaak** [comment]
@eduardo-rodrigues Another old PR just got merged by @henryiii (on fitting binned data).
Could a new version perhaps be released when this one is accepted? Thanks!

**Comment by eduardo-rodrigues** [comment]
> @eduardo-rodrigues Another old PR just got merged by @henryiii (on fitting binned data).
> Could a new version perhaps be released when this one is accepted? Thanks!

We can certainly help, and knowing that releases are for the benefit of actual use(rs) is motivating. But from https://github.com/scikit-hep/probfit/compare/1.2.0...master I only see a README update from me. Seems you already have what you need with Henry's release, no?

**Comment by henryiii** [comment]
1.2 does have that PR, yes.

File: setup.cfg [hunk 1]
```diff
@@ -6,7 +6,7 @@ long_description_content_type = text/x-rst
 url = https://github.com/scikit-hep/probfit
 author = Piti Ongmongkolkul
 author_email = piti118@gmail.com
-maintainer = Scikit-HEP
+maintainer = The Scikit-HEP admins
```
Comment (henryiii) [inline setup.cfg:9]: That's weird, I thought I copy & pasted...

**Review by henryiii — APPROVED** [review]

**Comment by mbaak** [comment]
I had missed release 1.2. Perfect, thanks!

**Comment by eduardo-rodrigues** [comment]
Morning @mbaak. BTW, do not hesitate to get in touch again in the future; and if you want to do so more generally there's the PyHEP (Python in HEP) community from the HSF, see https://hepsoftwarefoundation.org/workinggroups/pyhep.html and in particular the main communication channel https://gitter.im/HSF/PyHEP.

**Pull request merged**